<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlightSettingsPage</class>
 <widget class="QWidget" name="HighlightSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="localHighlightsIcon">
       <property name="text">
        <string notr="true">[icon]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="localHighlightsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Legacy Highlights are replaced by Highlights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="localHighlightsDetails">
       <property name="text">
        <string>Details...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Custom Highlights</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="highlightTable">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <column>
         <property name="text">
          <string>Enabled</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Highlight</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RegEx</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Channel</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Highlight Nicks</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="highlightAllNicks">
        <property name="text">
         <string>All nicks from identity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highlightCurrentNick">
        <property name="text">
         <string>Current nick</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highlightNoNick">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="nicksCaseSensitive">
        <property name="text">
         <string>Case sensitive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>highlightNoNick</sender>
   <signal>toggled(bool)</signal>
   <receiver>nicksCaseSensitive</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
